### PR TITLE
fix indents and sections headers in concepts

### PR DESCRIPTION
**Pull request by hbagdi**
Concepts docs has exploded with content that could use some
re-organization.

This is small patch in a series of fixes to make this document
understandable and easy to navigate.

### CODE REVIEW

**Comment by jpeach** [comment]
s/indendts/indents/, in the commit. Suggest that the commit fixes the section headers, not just indents

File: docs-src/concepts.md [hunk 1]
```diff
@@ -424,7 +426,7 @@ status entries corresponding to their name. Both `GatewayStatus` and
 This is a list that includes a type of condition, the status of that condition,
 and the last time this condition changed.
 
-#### Listeners
+##### Listeners
```
Comment (jpeach) [inline docs-src/concepts.md:429]: Why does `Listeners` nest inside `Gateway Status`. I would expect it to be at the same level since it's a spec field.

**Comment by hbagdi** [comment]
Updated. PTAL.

**Comment by robscott** [comment]
Thanks!

/approve

**Pull request merged**